packages/client/src/scripts/postinstall.ts

```diff
@@ -1,36 +1,34 @@
-// @ts-check
-const childProcess = require('child_process')
-const { promisify } = require('util')
-const fs = require('fs')
-const path = require('path')
-const c = require('./colors')
+import Debug from '@prisma/debug'
```

Comment:
This file logic is almost intact, it's mostly been converted to TS so that we can bundle it.